docs/settings.rst

```diff
@@ -28,7 +28,7 @@ by the :func:`matplotlib.colors.to_hex` function from Matplotlib::
     >>> wwt.constellation_figure_color = '#D3BC8D'
     >>> wwt.constellation_selection_color = (1, 0, 1)
 
-Numerical settings all take values as Astropy :class:`astropy.units.Quantity`
+Numerical settings all take values as astropy :class:`astropy.units.Quantity`
```

Comment:
Looking at the rest of the docs, it doesn't look like we're fully consistent about this, but Astropy should stay capitalized when it refers to the project and not the specific `astropy` Python module.

Reply:
Changed.